bee-hive/maestro

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```
I believe this is the current preferred approach as it looks for a version of bash in the path, which can differ between systems

Reply:
OK.